publish_to_hugging_face.py

```diff
@@ -0,0 +1,41 @@
+from app_utils import hugging_face_utils
+
+import argparse
+import logging
+import sys
+
+if __name__ == "__main__":
+  parser = argparse.ArgumentParser(description="")
+  parser.add_argument("-e", "--experiment_dir", required=True, help="experiment dir", default=argparse.SUPPRESS)
+  parser.add_argument("-d", "--device", help="'cpu' or 'cuda:0', if the GPU device id is 0", default=argparse.SUPPRESS)
+  parser.add_argument("-a", "--api_key", help="Hugging Face API Key", default=argparse.SUPPRESS)
+  parser.add_argument("-r", "--repository_id", help="Hugging Face Repository ID", default=argparse.SUPPRESS)
+
+  parser_args, unknown = parser.parse_known_args(sys.argv)
+
+  #args = vars(parser.parse_args())
+
+  if "experiment_dir" in parser_args:
+        experiment_dir = parser_args.experiment_dir
+  else:
+    raise ValueError("Please, provide the experiment directory")
+
+  if "device" in parser_args:
+        device = parser_args.device
+  else:
+    raise ValueError("Please, provide the device")
+
+  if "api_key" in parser_args:
```

Comment:
`api_key` can potentially be an optional argument, as uploading should work when logged into hf hub (haven't check this yet, though).